src/index.ts

```diff
@@ -84,22 +87,67 @@ const SchemaExtensionPlugin = makeExtendSchemaPlugin(build => {
           const {
             graphile: { fieldContext },
           } = resolveInfo;
-          return representations.map((representation: any) => {
+          return representations.map(async (representation: any) => {
```

Comment:
This needs a `Promise.all` wrapper
```suggestion
          return Promise.all(representations.map(async (representation: any) => {
```

Reply:
No it doesn't - ignore.